File: InJava/app/src/main/java/com/barryzeha/simpleweather/common/entities/Forecast.java
package com.barryzeha.simpleweather.common.entities;

/****
 * Project Simple Weather
 * Created by Barry Zea H. on 21/01/2023
 * Copyright (c)  All rights reserved.
 ***/

public class Forecast extends WeatherBase {
 long sunrise;
  double pop;


 public long getSunrise() {
  return sunrise;
 }

 public double getPop() {
  return pop;
 }

 public void setPop(double pop) {
  this.pop = pop;
 }


}

File: InKotlin/app/src/main/java/com/barryzeha/simpleweather/common/entities/Forecast.kt
package com.barryzeha.simpleweather.common.entities

/****
 * Project Simple Weather
 * Created by Barry Zea H. on 18/01/2023
 * Copyright (c)  All rights reserved.
 ***/
data class Forecast(
    val dt: Long,
    val humidity: Int,
    val temp: Double,
    val weather: List<Weather>,
    val sunrise: Long,
    val pop: Double
) : WeatherBase(dt, humidity, temp, weather)
